onnx/optimizer/pass_manager.h

```diff
@@ -35,7 +35,7 @@ class GeneralPassManager : public PassManager {
   std::shared_ptr<PassManagerAnalysis> run(Graph& graph) override;
 
  protected:
-  std::set<std::shared_ptr<Pass>> passes;
+  std::vector<std::shared_ptr<Pass>> passes;
```

Comment:
Maybe add a comment saying the order we add the passes matters, so people won't accidentally change it back to set.

Reply:
Sure, will add that